_posts/2021-01-13-an-introduction-to-mixing-audio.md

```diff
@@ -0,0 +1,42 @@
+
+This is where audio mixing software and hardware comes into play. Audio mixing allows us to properly route our audio to where it needs to be and often times gives it a boost in quality, while also making it easier to handle unlike windows sound settings and such. Having more options can be daunting at times, especially when just starting out, but once you get the hang of it, more often than not it saves so much more time and provides you with better quality than if you just went without.
+
+### - Audio Mixing options
+
+Are you interested in having more control over your audio? Lets go over our options then!
+
+###### - Mixing in OBS?
+You might say OBS has its own audio mix, that is true. However, just like using windows, it can be complicated. First you have to find audio levels in windows that suite yourself, so you can hear the game and other things. Then you have to pull the audio directly into OBS and try and manage it there. It can be a complete mess because the audio levels are already being adjusted for your hearing first. Optionally, if you adjust for the OBS Mix first, the levels for you hearing the audio will also be affected, too high, too low, and when you adjust the audio levels going to your headphones it will then adjust the levels in OBS. Big wonky mess right?
+
+###### - GoXLR, GoXLR-Mini, Elgato Wave
+ I'm sure every streamer has heard of a GoXLR, or GoXLR-Mini which come with software mixing alongside hardware controls. There is also the Elgato Wave microphones which come with similar software mixing and some hardware controls.
+
+###### - Physical Mixers
+Physical mixers are also available of various sizes and prices that stretch from less than $50 US Dollars to well over $500 to even $1000s. These beastly devices offer physical nobs and sliders for customization and take up a bit more space than a GoXLR but in some cases not by much or at all as some of them can be fairly small. However, their prices range from significantly lower than even a GoXLR-Mini or even cheaper then the Elgato Wave.
+
+###### - Voicemeeter
+Finally, there is Voicemeeter, something either mostly unheard of or often not spoken. Voicemeeter is a digital mixing software for windows designed to help thread and control audio similar to a physical mixer, except that it is an application ran on windows.
+
+Voicemeeter Banana Article: [Voicemeeter Banana]({% post_url 2021-01-14-voicemeeter-banana %})
+
+Coming Later: Physical Mixer, GoXLR/GoXLR-Mini, Elgato Wave.
```

Comment:
Instead of just saying "Coming Later", talk about what specifically is coming later and what to expect from these articles. You have a list of things saying "Coming Later", which is vague on what the reader should expect.

Reply:
instead of coming later:

Reply:
At some point we would like to include different types of audio setups. I believe OGJunkyard is currently doing a piece on his entire streaming setup which will include a very in-depth audio setup. I have two different sized physical mixers and would like to cover each one both individually and comparatively together explaining my reasons for the upgrade and reasons I could have kept my old one and would have been just fine. As stated above, we are interested in working with someone to get articles on the GoXLR, GoXLR-Mini and the Elgato Wave, so if interested please hit us up!